pkg/playground/react-demo-cards.jsx

```diff
@@ -0,0 +1,89 @@
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from "react";
+import ReactDOM from "react-dom";
+import "./react-demo.css";
+
+import {
+    Button, Card, CardHeader, CardBody, CardFooter,
+    CardHead, CardActions, Gallery, GalleryItem
+} from '@patternfly/react-core';
+
+const CardsDemo = () => {
+    const cards = [
+        <Card key="card0" className="flex-card" isCompact isHoverable>
+            <CardHeader>This is a compact card</CardHeader>
+            <CardBody>I'm also hoverable</CardBody>
+            <CardFooter>Compact cards have footers</CardFooter>
+        </Card>,
+        <Card key="card1" className="flex-card">
+            <CardHead>
+                <CardActions>
+                    <input type="checkbox" />
+                    <Button className="btn">click</Button>
+                </CardActions>
+                <CardHeader>This is a card header</CardHeader>
+            </CardHead>
+            <CardBody>This is a card body</CardBody>
+            <CardFooter>And here is my footer</CardFooter>
+        </Card>,
+        <Card key="card2" className="flex-card" isCompact isHoverable>
+            <CardHeader>This is a compact card</CardHeader>
+            <CardBody>I'm also hoverable</CardBody>
+            <CardFooter>Compact cards have footers</CardFooter>
+        </Card>,
+    ];
+    const galleryCards = [
+        <Card key="card3" isCompact>
+            <CardBody>I'm a card in a gallery</CardBody>
+        </Card>,
+        <Card key="card4" isCompact>
+            <CardBody>I'm a card in a gallery</CardBody>
+        </Card>,
+        <Card key="card5" isCompact>
+            <CardBody>I'm a card in a gallery</CardBody>
+        </Card>,
+        <Card key="card6" isCompact>
+            <CardHeader>I have a header too</CardHeader>
+            <CardBody>I'm a card in a gallery</CardBody>
+        </Card>,
+        <Card key="card7" isCompact>
+            <CardBody>I'm a card in a gallery</CardBody>
+        </Card>,
+    ];
+    let i = 0;
+
+    return (
+        <fragment>
+            <h4>PF4 cards arranged using flex</h4>
+            <div className="flex-cards">
+                { cards }
+            </div>
+            <h4>PF4 cards arranged using a PF4 Gallery</h4>
+            <Gallery gutter="md">
+                { galleryCards.map(c => <GalleryItem key={i++}>{c}</GalleryItem>) }
```

Comment:
This is not a legit usage of key=. The key needs to be an intrinsic property of the card, not such an external one, otherwise the whole purpose is moot.  The galleryCards array already has keys (even though they aren't needed), so how about recycling them as `key={"item" + ... }` here? The whole thing would be quite a bit cleaner if the `<GalleryItem>` wrapper would just be part of the array, so that you can (1) use it directly here without re-mapping, and (2) add the key= to the Item wrapper instead of the `Card`.

This may be a bit pedantic  for this use case, but as the purpose of this is to find out how to use them and provide an copy&paste example, let's be correct here.

I know this is prop